src/components/games/HomeTeamGameItem.js

```diff
@@ -5,7 +5,7 @@ const TeamGameItem = (props) => {
   return (
     <div className={classes.team}>
```

Comment:
ink it is better to create a common component for HomeTeamGameItem and AwayTeamGameItem.
the only difference between them is the location of the score.

You can create a shared component, with a prop to monitor the location of the score.